src/current/_data/versions.csv

```diff
@@ -13,4 +13,4 @@ v22.1,2022-05-24,2023-05-24,2023-11-24,v21.2,release-22.1
 v22.2,2022-12-05,2023-12-05,2024-06-05,v22.1,release-22.2
 v23.1,2023-05-15,2024-05-15,2024-11-15,v22.2,release-23.1
 v23.2,2024-02-05,2025-02-05,2025-08-05,v23.1,release-23.2
-v24.1,N/A,N/A,N/A,v23.2,main
+v24.1,N/A,N/A,N/A,v23.2,master

```

Comment:
Confirming that this is meant to be the `cockroachdb/cockroach` branch, not the docs branch.